protocols/migration/migration_reports_5_0_0_to_reports_4_0_0.py

```diff
@@ -0,0 +1,174 @@
+import logging
+
+from protocols import reports_4_0_0 as reports_4_0_0
+from protocols import reports_5_0_0 as reports_5_0_0
+from protocols.migration.base_migration import BaseMigration
+from protocols.migration.base_migration import MigrationError
+
+
+class MigrateReports500To400(BaseMigration):
+
+    old_model = reports_5_0_0
+    new_model = reports_4_0_0
+
+    def migrate_clinical_report_rd(self, old_instance):
+        """
+        :type old_instance: reports_5_0_0.ClinicalReportRD
+        :rtype: reports_4_0_0.ClinicalReportRD
+        """
+        new_instance = self.convert_class(self.new_model.ClinicalReportRD, old_instance)  # :type self.new_model.ClinicalReportRD
+
+        # type of interpretationRequestVersion has been changed from int to str
+        new_instance.interpretationRequestVersion = str(old_instance.interpretationRequestVersion)
+
+        # references has been renamed to supportingEvidence
+        new_instance.supportingEvidence = old_instance.references
+
+        # converts all reported variants
+        new_instance.candidateVariants = [
```

Comment:
If `old_instance.variants==None` then this will fail. Every nullable list needs to be checked for this.

Reply:
Thanks @priesgo -- I've added the changes to https://github.com/genomicsengland/GelReportModels/pull/266/commits/8643d405a83939160a61c36619ad8fc4949682fb